app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt

```diff
@@ -234,28 +231,6 @@ class TopicRevisionFragmentTest {
     }
   }
 
-  @Test
-  // TODO(#973): Fix TopicRevisionFragmentTest
-  @Ignore
-  fun testTopicRevisionFragment_loadFragment_configurationChange_checkSpanCount_isThree() {
```

Comment:
Why was this removed?

Reply:
Shifted to `ProfileChooserSpanTest` as all the test related to span is there.

Reply:
``ProfileChooserSpanTest`` is testing profile span wrapping, not the number of spans for the revision card. Don't we still need this? E.g. the profile span functionality might change--that won't affect how we handle the revision card columns.

Reply:
ah, my bad, I am sorry here, this test is shifted to `TopicRevisionSpanTest`

Reply:
Aha, thanks for clarifying. Got it.